src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx

```diff
@@ -106,14 +145,18 @@ const ManageYourVAHealthCare = ({
     >
       <PrescriptionsWidget />
     </DowntimeNotification>
-    {isEnrolledInHealthCare && <ScheduleAnAppointmentWidget />}
+    {isEnrolledInHealthCare &&
+      !isCernerOnlyPatient && <ScheduleAnAppointmentWidget />}
+    {isEnrolledInHealthCare &&
+      isCernerOnlyPatient && <ScheduleAnAppointmentCernerWidget />}
   </>
 );
 
 const mapStateToProps = state => {
   const isEnrolledInHealthCare = isEnrolledInVAHealthCare(state);
   const hcaEnrollmentStatus = selectEnrollmentStatus(state);
   const showServerError = hasESRServerError(state);
+  const isCernerOnlyPatient = selectIsCernerPatient(state);
```

Comment:
sorry just saw that this prop should be renamed to `isCernerPatient` for clarity.